source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,245 @@
+- Must have storage auto-scaling enabled
+- Must be M10+ to access this setting
+- Must have time-based oplog setting enabled
+
+Client Apps:
+
+- ``realm-core`` vTBD+, which means these SDK versions: TBD
+
+.. _realm-sync-migrate-what-to-know:
+
+What to Know Before migrating
+-----------------------------
+
+Migrating your App's Sync mode involves changes that affect both client apps and
+your App's backend. You should be aware of the side effects and plan ahead.
+
+How long a migration takes is directly proportional to the amount of data that
+needs to be migrated. The more data, the longer it will take to migrate.
+
+Migrations are not permenant immediately after completion. They can be canceled
+during the migration or rolled back within a certain timeframe after the migration
+is finished.
+
+Progress notifications will not work after migration because Flexible Sync does
+not yet support them.
+
+Client apps that use an SDK with with a realm-core version lower than TBD will
+experience a client reset after migration. You should use a 
+:ref:`client reset <handle-a-client-reset>` handler with recovery enabled.
+
+.. _realm-sync-migrate-stages:
+
+Migration Stages
+~~~~~~~~~~~~~~~~
+
+There are three stages to a Sync mode migration. You can see your migration's
+stage in the App Services UI.
+
+#. Disabling Partition-Based Sync
+   
+   At this stage, client apps can connect to your App's backend, but local writes 
+   will not sync. These writes will be lost if you don't have a 
+   :ref:`client reset handler <handle-a-client-reset>` to recover unsynced changes.
+	 
+   Clients can also receive updates to MongoDB data, but they won't
+   be able to recieve upload acknowledgements.
+
+#. Initializing Flexible Sync
+   
+   Clients cannot connect at this stage. The time this takes is directly
+   proportional to the amount of data in synced collections.
+
+#. Migration completed
+
+   **Manage sync migration** appears on Sync page if the migration hasn't been 
+   committed or rolled back. This button lets admins commit or roll back manually.
+
+Migrate Partition-Based Sync App to Flexible Sync
+-------------------------------------------------
+
+After beginning migration from Partition-Based Sync to Flexible Sync, the process
+is mostly automatic. However, you can control important parts of the migration.
+
+During a migration, you can cancel it. After a migration, you can roll it back 
+or manually commit the migration.
+
+Begin Migration
+~~~~~~~~~~~~~~~
+
+Before you begin a migration, make sure you're aware of the
+:ref:`migration stages <realm-sync-migrate-stages>` and
+:ref:`effects <realm-sync-migrate-what-to-know>`.
+
+During migration, client apps may experience side effects, most of which are similar
+to what happens if you disable and re-enable sync. These are some additional
+side effects of migration:
+
+- In the App Services UI, the Rules and Schemas pages will be unresponsive.
```

Comment:
@kmorkos, do you think "unresponsive" is a fair description of what will happen? I haven't seen that side effect.

Reply:
It wouldn't be unresponsive, just read-only. So they can still see their schemas (I think there's a bug that might lead to a broken view of rules), but if they try to make any changes to their config it will fail

Reply:
Ah, that makes a lot more sense. Thank you!